Added portuguese language

RenanMsV opened the pull request.

RenanMsV committed:
added portuguese language

RenanMsV commented:
Brazilian Portuguese *

Southclaws commented on components/languages.js:
```diff
@@ -3,7 +3,8 @@ export const LANGUAGES = {
   "🇩🇪": "German",
-  "🇷🇺": "Russian"
+  "🇷🇺": "Russian",
+  "PT": "Portuguese"
```
🇧🇷

Southclaws replied:
Copy-paste that glyph - I'll replace these with SVGs at some point.

RenanMsV replied:
Fixed conflicts, and added the correct font type brazil . Changed to BR since Brazilian Portuguese isnt the same as Portuguese. If you want to change to PT just request changes again.

RenanMsV committed:
Merge branch 'master' into portuguese

RenanMsV commented:
Fixed conflicts, and added the correct font type 🇧🇷 . Changed to BR since Brazilian Portuguese isnt the same as Portuguese. If you want to change to PT just request changes again.

Southclaws commented:
Awesome, thank you! :)
